pkg/urlutil/git_test.go

```diff
@@ -9,6 +9,7 @@ var (
 		"git@bitbucket.org:atlassianlabs/atlassian-docker.git",
 		"https://github.com/docker/docker.git",
 		"http://github.com/docker/docker.git",
+		"http://github.com/docker/docker.git#branch",
```

Comment:
`#.+` means `whatever after #` there is no need to test `:dir` because that already whatever.